modules/monitoring-managing-core-platform-alerting-rules.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: CONCEPT
+[id="managing-core-platform-alerting-rules_{context}"]
+= Managing alerting rules for core platform monitoring
+
+:FeatureName: Creating and modifying alerting rules for core platform monitoring
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+{product-title} {product-version} monitoring ships with a large set of default alerting rules for platform metrics.
+As a cluster administrator, you can customize this set of rules in two ways:
+
+* Modify the settings for existing platform alerting rules by adjusting thresholds or by adding and modifying labels.
+For example, to help you route and triage issues flagged by an alert, you can change the `severity` label for an alert from `warning` to `critical`.
+
+* Define and add new custom alerting rules by constructing a query expression based on core platform metrics in the `openshift-monitoring` namespace.
+After this new, user-defined rule exists, you can view, edit, and remove it.
```

Comment:
`After this new, user-defined rule exists`
I am kindly think the expression is not good, but don't know what's the better expression

Reply:
I've deleted this sentence. It doesn't add any useful information that a user doesn't already know,